CorsixTH/Lua/rooms/training.lua

```diff
@@ -212,9 +212,8 @@ function TrainingRoom:commandEnteringStaff(humanoid)
         humanoid.last_room = nil
       end
     end
-  elseif humanoid.humanoid_class ~= "Handyman" then
-    self.world.ui.adviser:say(_A.staff_place_advice.only_doctors_in_room
-    :format(_S.rooms_long.training_room))
+  elseif humanoid.humanoid_class == "Nurse" then
```

Comment:
Receptionist...

Reply:
They can't be placed in rooms, so I was aiming for a more descriptive check.